customer-service-tests/EmployeeModelTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Xunit;
+using System.Collections.Generic;
+using customer_service;
+using customer_service.Models;
+using customer_service.Data;
+
+namespace customer_service_tests
+{
+    public class EmployeeModelTests
+    {
+        [Fact]
+        public void CanSaveNewEmployeeToDatabase()
+        {
+            var employeeFactory = new EmployeeFactory();
+            List<Employee> allEmployees = employeeFactory.getAll();
+            int newEmployeeId = allEmployees[allEmployees.Count - 1].EmployeeId;
+
+            Employee Jeb = new Employee()
+            {
+                FirstName = "Jeb",
+                LastName = "Bush",
+                DepartmentId = 1,
+                Administrator = false
+            };
+
+            Jeb.save();
+
+            var shouldBeJeb = employeeFactory.get(newEmployeeId);
+
+            Assert.NotNull(shouldBeJeb);
+            Assert.True(newEmployeeId == shouldBeJeb.EmployeeId);
```

Comment:
Here we just need to assert that EmployeeId is not null:
`Assert.NotNull(shouldBeJeb.EmployeeId)`